Assets/MixedRealityToolkit/_Core/Devices/BaseController.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Devices
+{
+    /// <summary>
+    /// Base Controller class to inherit from for all controllers.
+    /// </summary>
+    public class BaseController : IMixedRealityController
```

Comment:
I still think it's easier to add visibility than to take it away once we release (even an Alpha).

Reply:
Except it can / probably will cause issues in testing while we are trying to build the MVP

It's far easier to set a specific task to reduce visibility when we have a working test model later.  So we can create that task for either Pre-Alpha or pre-Beta

Reply:
Agreed we'll make them internal but for now we should be the only ones working in the repository.

Reply:
Once we figure out how this will go, we'll update the accessors.